server/routes/fish.js

```diff
@@ -17,6 +17,12 @@ router.get('/:id', (req, res) => {
     .then(fishDetail => res.status(200).json(fishDetail))
 })
 
+router.put('/:id', (req, res) => {
+  const fishId = Number(req.params.id)
+  fishDetails.where('fishId', fishId)
```

Comment:
How about using your db function you created in `server/db/fish` to not duplicate the code?